uint8 stack_status

# stack_status enum values
uint8 INITIAL=0
uint8 START=1
uint8 RUNNING=2
uint8 EXIT=3